TeamMandate/Presentation.md

```diff
@@ -10,112 +10,80 @@ As a student who isn't taking any courses this summer, I decided to look for wor
 
 ... 
 
-## The How
+## Proposed IITB Mandated Priorities
 
-- Do not compete with private sector 
-  - Integration over development 
-  - Create jobs, not software 
+### Avoid Rework 
 
->delivering programs that help Canadians move through life’s transitions, from school to work, from one job to another, *from unemployment to employment* and from the workforce to retirement; 
+Work in the open
 
-- Avoid rework 
-  - Work in the open 
+#### Repercussions
+
+- All teams (Ex: developers, security and testing) must use git
+
+#### Opportunities
+
+- Visibility will allow teams to understand an always up to date version of the processes of other teams 
+- Reduce rework from sharing and building on the work of others 
+  - Example: CDS implements a secure document uploading project. CSE 
 
 >Values and Ethics Code for the Public Sector: 
 Federal public servants are entrusted to use and care for public resources responsibly, for both the short term and long term.
 
-- Modern tools 
-  - Fully separated client and back ends 
-  - All communications through service interfaces
-  - Communications between teams must be through these interfaces
+... 
+
+### Modern tools 
 
 >Digital Operations Strategic Plan: 2018-2022
 - public servants are more effective at delivering services, programs and policies when they have modern tools that help their work and a modern workplace
 - public servants need professional development that keeps up with changing times and changing technology
 
-## Actions 
-
-### Creation of Umbrella Fund 
-
-#### Vision
-
-Allow the creation of new agile procedures for approval, software, and hardware onboarding which self manages all resource sand funding with the intent of trialing new ways of working for the benefit of the organization. 
-
-#### Implementation 
-
-##### Onboarding 
-
-Through collaboration with the existing Innovation Hub, onboard new projects and ideas from the business lines 
-
-##### Funding 
+#### Repercussions
 
-*Option 1*
+- Blanket approve all future versions of widely supported and popular tools from trusted organizations 
+  - Ex: Google Chrome, Google Docs, Google Hangouts, YouTube, Facebook, Twitter, Firefox, Slack, node, npm
 
-Engage Innovation Hub about possible funding avenues 
+#### Opportunities
 
-*Option 2*
+- Requiring rigorous review and approval of tools enforces the use of dated tools which 
+  - Reduce the ability for the workforce to complete their work 
+  - Introduce security concerns by not keeping tooling up to date 
```

Comment:
change to
- Focus security activities on resiliency (e.g. keeping tooling up to date, enable rapid response to cyber incidents) instead of approving new tools